tests/page_objects/providers_care_platform_page.yaml

```diff
@@ -68,6 +69,8 @@ providers_facilities_page:
   local_hospitals_button: //button[text()='Local Hospitals']
   devices_tab: //button[text()='Devices']
   title_subtab_local_hospital: //h4[text()='Local Hospitals & ED Contact']
+  call_history_accordion: (//div[@class='accordion box-scrollable-content pl -ml text-sm']/div[starts-with(@class, 'accordion-item')])
```

Comment:
Could you use only the parts of the class that matters? all that part of the class ("pl -ml text-sm") increase the probabilities of getting this selector to be obsolete. Also, you don't need to wrap all inside of parenthesis if you are not going to use an index (Ex: [1]) or function(Ex: [last()]) that needs to work with the rest of the selector as a whole.

Reply:
@dguzmanneural is there perhaps another way recommended to find the element? Without that part of the class, the element is not located in the test and it doesn't have many unique attributes. 

![image](https://github.com/neveralone-chs-org/TestRay/assets/110423707/8938a67c-929f-44b1-9ac8-cb590a9e65c4)